pwnlib/gdb.py

```diff
@@ -236,13 +236,14 @@ def _gdbserver_args(pid=None, path=None, args=None, which=None, env=None):
     if pid:
         gdbserver_args += ['--once', '--attach']
 
-    if env:
+    if env != None:
```

Comment:
Identity comparison is faster and generally recommended by Python docs when dealing with `None`:
```suggestion
    if env is not None:
```